affinity/cpp-20/d0796r2.md

```diff
@@ -170,150 +187,134 @@ This feature could be easily scaled to heterogeneous and distributed systems, as
 
 In this paper we propose an interface for querying and representing the execution resources within a system, queurying the relative affinity metric between those execution resources, and then using those execution resources to allocate memory and execute work with affinity to the underlying hardware. The interface described in this paper builds on the existing initerface for executors and execution contexts defined in the executors proposal [[22]][p0443r4].
 
-### Execution resources
+### Terminology
 
-An `execution_resource` is a light weight structure which acts as an identifier to particular piece of hardware within a system. It can be queried for whether it can allocate memory via `can_place_memory` and whether it can execute work via `can_place_agents`, and for it's name via `name`. An `execution_resource` can also represent other `execution_resource`s, these are refered to as being *members of* that `execution_resource` and can be queried via `resources`. Additionally the `execution_resource` which another is a *member of* can be queried vis `member_of`. An `execution_resource` can also be queried for the concurrency it can provide; the total number of *threads of execution* supported by that *execution_resource* and all resources it represents.
+An **execution agent** executes work, typically implemented by a *callable*, 
+on an **execution resource** of a given **execution architecture**. 
+An **execution context** manages a set of execution agents on an 
+execution resource.
+An **executor** submits work to an execution context.
+More that one executor may submit work to an execution context.
+More than on execution context may manage execution agents
+on an execution resource.
 
-> [*Note:* Note that an execution resource is not limited to resources which execute work, but also a general resource where no execution can take place but memory can be allocated such as off-chip memory. *--end note*]
```

Comment:
I think this note has been lost in the changes.

Reply:
Defines a memory resource vs. an execution resource.

Reply:
That's right, ignore me :)